tests/system/model_monitoring/test_model_monitoring.py

```diff
@@ -1387,19 +1386,20 @@ def _train(cls) -> None:
             cls.y_train,  # pyright: ignore[reportGeneralTypeIssues]
         )
 
-    def _get_monitoring_feature_set(self) -> mlrun.feature_store.FeatureSet:
+    def _get_monitoring_feature_set(
+        self, endpoint_id: str
+    ) -> mlrun.feature_store.FeatureSet:
         model_endpoint = mlrun.get_run_db().get_model_endpoint(
```

Comment:
A more proper approach would be passing the feature set uri when calling `_test_feature_names` from `test_inference_feature_set` (because you have the mep record at that point) and then you could retrieve it directly without getting the mep from the db again. Anyway it's not mandatory for now